src/p2p/ServiceQueue.ts

```diff
@@ -727,6 +824,78 @@ function sortedInsert(
   }
 }
 
+function verifyDebugDropNGT(reqParamsDropNGT, cycle): number {
```

Comment:
It would be better if this could return a object like `{'success': true/false, reason: ''}` instead of arbitrary numerical response.  it will simplify the usage of this function to not have to check each response (0,1,2) when you can instead check if (response) and log out reason if needed.

Reply:
good point. pushed changes